.ci.yaml

```diff
@@ -106,9 +106,12 @@ platform_properties:
           {"dependency": "gems"},
           {"dependency": "ios_signing"}
         ]
-      os: Mac-10.15
-      device_os: iOS-14.4.2
-      xcode: 12c33
+      # TODO (mohsinally@): Remove the multi-values in 
+      #  dimensions below once the fleet is all upgraded.
+      #  https://github.com/flutter/flutter/issues/94767
+      os: Mac-10.15|Mac-12.0
+      device_os: iOS-14.4.2|iOS-15.1
+      xcode: 12c33|13a233
```

Comment:
This shouldn't have the `|` as it's a test level construct

Reply:
Should it just be a space?

Reply:
Does dimension matching support ORs?